### ISSUE TITLE
Breaking change in aiohttp-0.18.0

### ISSUE THREAD

**Issue by avanov** [aio-libs/aiohttp_debugtoolbar#32]
Related to the following change - https://github.com/KeepSafe/aiohttp/pull/567

The version ``0.0.5`` of the package fails on the following step:
https://github.com/Axik/aiohttp/blob/a2aa20dacbeb34544cc69765e20dabe5718c0d57/aiohttp/web_urldispatcher.py#L448

**Comment by asvetlov** [comment]
Please provide an example of name what you use in your code.

**Comment by avanov** [comment]
It has to do with the following pre-defined names - https://github.com/aio-libs/aiohttp_debugtoolbar/blob/master/aiohttp_debugtoolbar/main.py#L62-L89 (dot symbols, I suppose).

My app is just using the middleware:
```python
middleware = [aiohttp_debugtoolbar.middleware]
app = web.Application(handler_factory=web.RequestHandlerFactory,
                      middlewares=middleware)
aiohttp_debugtoolbar.setup(app)
```

**Comment by asvetlov** [comment]
Got it. Will publish bugfix release allowing dots and columns today.

**Comment by avanov** [comment]
Thanks!

**Comment by asvetlov** [comment]
Fixed by aiohttp 0.18.1 release
